src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -38,29 +38,90 @@ To provide a good user experience, **sites should strive to have an LCP of 2.5 s
+
+### Using PageSpeed Insights CrUX supplementary metrics
+
+As well as LCP, those looking to optimize LCP, should also use the [First Contentful Paint (FCP)](/fcp/) and [Time to First Byte (TTFB)](/ttfb/) timings which can provide valuable insights into LCP.
+
+TTFB is the time when the visitor starting to navigate to a page (for example, clicking on a link), until the first bytes of the HTML document are received. A high TTFB can make achieving a 2.5 second LCP challenging, or even impossible.
+
+A high TTFB can be due to mutiple server redirects, visitors located far away from the nearest site server, visitors on poor network conditions, or an inability to use cached content due to query parameters.
+
+Once a page starts rendering there may be an initial paint (for example, the background color is painted), followed by some content showing up (for example, the site header). The appearnce of the initial conent is measured by FCP. The delta between FCP and other metrics can be very telling.
+
+A large delta between TTFB and FCP could indicate that the browser needs to download a lot of render-blocking assets. It can also be a sign it must complete a lot of work to render any meaningful content—a classic sign of a Client Side Rendered site.
+
+A large delta between FCP and LCP indicates that the LCP resource is either not immediately available for the browser to prioritise (for example, text or images that are managed by JavaScript rather than being available in the initial HTML) or that the browser is completing other work before it can display the LCP content.
+
+### Using PageSpeed Insights Lighthouse data
+
+The Lighthouse section of PageSpeed Insights offers some guidance to improving LCP, but first you should check if the LCP given is broadly inline with real user data provided by CrUX.
+
+If Lighthouse is showing no LCP issue, but the CrUX data is, then any Lighthouse suggestions may not be relevant. The opposite is also true—if Lighthouse is showing a really poor LCP time, but CrUX data is showing your users mostly have a good LCP, then it may not be necessary to further improve LCP. Just be sure to check that the CrUX data is for this page and not for the full origin as detailed above.
+
+If both sources of data are showing an LCP that should be improved than the Lighthouse section can provide valuable guidance on ways to improve LCP. Use the LCP filter to only show audits relevant to LCP:
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/himAvC1GMr7K0kcbvG4F.png", alt="Lighthouse LCP Opportunities and Diagnostics", width="800", height="498" %}
+
+As well as the Opportunities to improve, there is Diagnostic information that may provide more information to help diagnose the issue. The **Largest Contentful Paint elment** diagnostic shows a useful breakdown of the various timings that made up the LCP:
```

Comment:
Why is "Opportunities" and "Diagnostic" capitalized? Are they user interface elements in Lighthouse? If so, they should be in bold.